server/neptune/workflows/internal/prrevision/workflow.go

```diff
@@ -119,19 +125,49 @@ func (r *Runner) setRevision(ctx workflow.Context, req Request, prs []github.Pul
 	return nil
 }
 
+func (r *Runner) listModifiedFilesAsync(ctx workflow.Context, req Request, prs []github.PullRequest) map[github.PullRequest]workflow.Future {
+	now := workflow.Now(ctx)
+	taskQueue := TaskQueue
+	futuresByPullNum := map[github.PullRequest]workflow.Future{}
+
+	oldPrsCount := 0
+	for _, pr := range prs {
+		// if new version of workflow and pr is not updated within ThroughputCutOffDays, change the tq
+		version := workflow.GetVersion(ctx, version.MultiQueue, workflow.DefaultVersion, 1)
+		if version != workflow.DefaultVersion && !r.isPrUpdatedWithinDays(now, pr, r.SlowProcessingCutOffDays) {
+			taskQueue = SlowTaskQueue
+			oldPrsCount++
+		}
+
+		futuresByPullNum[pr] = r.GithubActivityExecutor.GithubListModifiedFiles(ctx, taskQueue, activities.ListModifiedFilesRequest{
+			Repo:        req.Repo,
+			PullRequest: pr,
+		})
+	}
+
+	newPrsCount := len(prs) - oldPrsCount
+	r.Scope.SubScope("open_prs").Counter(fmt.Sprintf("less_than_%d_days", r.SlowProcessingCutOffDays)).Inc(int64(newPrsCount))
```

Comment:
Just increment the stat in the loop with 1 each time so you don't have create variables for old and new.